### DISCUSSION TITLE
Need idea or solution to get live feed frames into slint-ui from v4l or gstreamer

### DISCUSSION THREAD

**Discussion by MohaBeacon** [slint-ui/slint#2528] [Q&A]
am using linux dev board, using rust for few days now. hard to get the image data type matching with opencv or others and also these data types into slint ui. Can any one suggest a way to proceed on this

**Comment by tronical** [comment]
I plan on picking up #977 again in a few weeks. (I know, we're late with this :(. Meanwhile, your best option is to retrieve the frames in RGB and create a `slint::Image` for each frame that you receive.
